test/e2e/utils.go

```diff
@@ -292,10 +293,24 @@ func smokeTestProductionExample(name, yamlFileName string) {
 	queryDeploymentName := name + "-query"
 	collectorDeploymentName := name + "-collector"
 
-	err := e2eutil.WaitForDeployment(t, fw.KubeClient, namespace, queryDeploymentName, 1, retryInterval, timeout)
+	if name == "simple-streaming" {
```

Comment:
Not critical, but wondering if there is a better way to identify that this is a `streaming` strategy test, rather than just based on the deployment name?